.github/workflows/aws-deploy.yaml

```diff
@@ -46,6 +46,8 @@ jobs:
         run: npm install
       - name: Generate AWS CDK bootstrap template
         run: npm run generate-cdk-bootstrap-template
+      - name: Add EnableKeyRotation property to CDK assets key
+        run: sed -i '/.*KeyPolicy/i\ \ \ \ \ \ EnableKeyRotation:\ true' cdk-bootstrap.yaml
```

Comment:
the previous step runs `npm run generate-cdk-bootstrap-template`,  generate-cdk-bootstrap-template in package.json executes `npx cdk bootstrap --show-template > org-formation/200-baseline/cdk-bootstrap.yaml`.  the cdk-bootstrap file is in a subdirectory not in the current directory.  therefore you probably want `sed -i '/.*KeyPolicy/i\ \ \ \ \ \ EnableKeyRotation:\ true' ./org-formation/200-baseline/cdk-bootstrap.yaml/cdk-bootstrap.yaml` ?

i think modifying a json might be easier than yaml file because you don't need to worry about the spacing and there are other tools like `jq` to change it.   if you want the previous step to generate a json just  modify the command to do this.. `npx cdk bootstrap --json --show-template > org-formation/200-baseline/cdk-bootstrap.json`